erpnext/stock/doctype/warehouse/warehouse.py

```diff
@@ -70,6 +73,53 @@ def on_trash(self):
 		self.update_nsm_model()
 		self.unlink_from_items()
 
+	def warn_about_multiple_warehouse_account(self):
+		"If Warehouse value is split across multiple accounts, warn."
+
+		def get_accounts_where_value_is_booked(name):
+			sle = frappe.qb.DocType("Stock Ledger Entry")
+			gle = frappe.qb.DocType("GL Entry")
+			ac = frappe.qb.DocType("Account")
+
+			return (
+				frappe.qb.from_(sle)
+				.join(gle)
+				.on(sle.voucher_no == gle.voucher_no)
+				.join(ac)
+				.on(ac.name == gle.account)
+				.select(gle.account)
+				.distinct()
+				.where((sle.warehouse == name) & (ac.root_type == "Asset"))
```

Comment:
the `root_type` condition i'm not sure about. AC Receivables also comes under assets. I thought of using Account Type but Ive noticed a lot of people don't set it
```suggestion
				.where((sle.warehouse == name) & (ac.root_type == "Asset"))
```

cc: @ankush

Reply:
It should be similar to this? 

https://github.com/frappe/erpnext/blob/98b28abde0cdca40c48600b4f579494924f7f9cc/erpnext/accounts/utils.py#L1272-L1298 

`account_type == "Stock"`